pkg/apis/jvmbuildservice/v1alpha1/dependencybuild_types.go

```diff
@@ -13,10 +13,7 @@ const (
 )
 
 type DependencyBuildSpec struct {
-	SCMURL  string `json:"scmURL,omitempty"`
-	SCMType string `json:"scmType,omitempty"`
-	Tag     string `json:"tag,omitempty"`
-	Path    string `json:"path,omitempty"`
+	ScmInfo SCMInfo `json:"scmInfo,omitempty,inline"`
```

Comment:
```suggestion
	SCMInfo SCMInfo `json:",inline"`
```

I've never seen folks inline the way you had it.  Did you try it with just inline first and it did not work for some reason?

Reply:
ah, that would be why it was not working